openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityThreadPanel/ActivityThreadPanelBody.tsx

```diff
@@ -251,10 +251,13 @@ const ActivityThreadPanelBody: FC<ActivityThreadPanelBodyProp> = ({
           />
         )}
         {isTaskType && (
-          <div className="tw-flex tw-justify-end tw-mr-2 tw-mt-2">
+          <Space
+            align="center"
+            className="w-full justify-end p-r-xs m-t-xs"
```

Comment:
After replacing it to Space we still need this classes?

Reply:
@chirag-madlani  yes, as to align it to the right-hand side!